recipe/patches/0005-Remove-all-dependencies-from-setup.py.patch

```diff
@@ -1,19 +1,17 @@
-From dc32b341c09175ae0e5bf77186d1e1279c533ef8 Mon Sep 17 00:00:00 2001
-From: Kai Fricke <kai@anyscale.com>
-Date: Fri, 3 Dec 2021 10:55:23 -0800
-Subject: [PATCH 5/5] Remove all dependencies from setup.py
+From d15934405a7f9f5234e2b612dd158610c4fa0261 Mon Sep 17 00:00:00 2001
+From: Tim Paine <3105306+timkpaine@users.noreply.github.com>
+Date: Mon, 25 Sep 2023 18:29:15 +0000
+Subject: [PATCH] Remove all dependencies from setup.py
 
-Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
-Signed-off-by: Matti Picus <matti.picus@gmail.com>
```

Comment:
This should not change, though you can add your `Signed-off-by`.

Also, the patch should ideally be generated with `git format-patch --no-signature`